packages/data-point-service/lib/cache-middleware.js

```diff
@@ -97,10 +67,18 @@ function handleRevalidateError (entityId, entryKey) {
       entryKey,
       error
     )
-    // there is really nothing to do here, on another iteration
-    // params.staleWhileRevalidate could take an object that has a
-    // custom error handler
-    return false
+
+    // remove revalidation flags to allow a new revalidation to happen
+    // if there is an error at this point we might not want to bubble it up
```

Comment:
"if there is an error at this point **we might not want** to bubble it up" - why or why not?